corehq/ex-submodules/pillowtop/es_utils.py

```diff
@@ -102,15 +92,9 @@ def meta(self):
             ES_INDEX_SETTINGS.get(settings.SERVER_ENVIRONMENT, {}).get(self.hq_index_name, {})
         )
 
-        overrides = copy(ES_ENV_SETTINGS)
         if settings.ES_SETTINGS is not None:
-            overrides.update({settings.SERVER_ENVIRONMENT: settings.ES_SETTINGS})
-
-        for hq_index_name in ['default', self.hq_index_name]:
-            for key, value in overrides.get(settings.SERVER_ENVIRONMENT, {}).get(hq_index_name, {}).items():
-                if value is REMOVE_SETTING:
```

Comment:
Don't we still want this `REMOVE_SETTING` logic? I think its intent is to let environments remove certain settings from the defaults in `ES_INDEX_SETTINGS`.

Reply:
I wasn't sure if we wanted to keep this. I think you added it a while ago so if you're happy to remove I'll get rid of it.

Reply:
sorry, mis-understood you're comment. I'll add this back.

Reply:
7b392463dac

Reply:
Thanks!